test/custom_extension_test.dart

```diff
@@ -0,0 +1,35 @@
+library expected_output.test.custom_extension_test;
```

Comment:
I was going to ask why there's a library definition here (since we've generally moved away from them) but then remembered that this is part of the reflective test discovery story.  Is that right? Assuming it is, maybe we could document that in the README more explicitly.